config/initializers/5_telemetry.rb

```diff
@@ -46,83 +46,4 @@
 telemetry.add_counter(:authorizer_cache_records_fetched, 'Number of records fetched by authorizer cache', [:class])
 
 # To decrease amount of metrics, labels must be allowed explicitly
-allowed_labels = {
-  controller: [
-    'api/v2/config_reports_controller',
-    'api/v2/hosts_controller',
-    'api/v2/puppet_hosts_controller',
-    'config_reports_controller',
-    'dashboard_controller',
-    'fact_values_controller',
-    'hostgroups_controller',
-    'hosts_controller',
-    'notification_recipients_controller',
-  ],
-  action: [
-    'index',
-    'show',
-    'create',
-    'update',
-    'destroy',
-    'export',
-    'generate',
-    'facts',
-  ],
-  class: [
-    'Architecture',
-    'AuthSource',
-    'Bookmark',
-    'CommonParameter',
-    'ComputeResource',
-    'ConfigReport',
-    'Domain',
-    'DomainParameter',
-    'Environment',
-    'FactName',
-    'FactValue',
-    'Filter',
-    'Filtering',
-    'GroupParameter',
-    'Host::Base',
-    'Host::Managed',
-    'Hostgroup',
-    'HostParameter',
-    'Image',
-    'JobTemplate',
-    'Location',
-    'LocationParameter',
-    'LookupKey',
-    'LookupValue',
-    'Model',
-    'Nic::Base',
-    'Nic::BMC',
-    'Nic::Bond',
-    'Nic::Bridge',
-    'Nic::Interface',
-    'Nic::Managed',
-    'Operatingsystem',
-    'Organization',
-    'OrganizationParameter',
-    'OsParameter',
-    'Parameter',
-    'ProvisioningTemplate',
-    'Ptable',
-    'PuppetclassLookupKey',
-    'Realm',
-    'RemoteExecutionFeature',
-    'Report',
-    'Role',
-    'SmartProxy',
-    'Subnet',
-    'SubnetParameter',
-    'Taxonomy',
-    'Template',
-    'TemplateInput',
-    'TemplateInvocation',
-    'TemplateKind',
-    'User',
-    'Usergroup',
-    'Widget',
-  ],
-}
-telemetry.add_allowed_tags!(allowed_labels)
+telemetry.add_allowed_tags!(SETTINGS[:telemetry][:allowed_labels])
```

Comment:
This is slightly safer: if `SETTINGS[:telemetry]` is `nil` it won't error out.
```suggestion
telemetry.add_allowed_tags!(SETTINGS.dig(:telemetry, :allowed_labels))
```